### DISCUSSION TITLE
Wii games or just game cube?

### DISCUSSION THREAD

**Discussion by NepsAndHelis** [darthcloud/BlueRetro#700] [Q&A] [answered]
I was just wondering about the GameCube adapter plugged into the wii. Would it be able to play wii games or just games or just GC?

**Comment by darthcloud** [comment, marked as answer]
You can do the same things a regular GC controller can do.

Most Wii games don't support GC controllers.

**Reply by SinisterSticks** [reply]
So it there going to be a Blue Retro Wii Classic Nunchuk hole adapter?  This would be so that my Xbox Adaptive Controller can be used with Wii Classic Controller games.
